modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,275 @@
+Anypoint Runtime Fabric is a package of components that work together, and updates are delivered in different ways:
 
-* Cluster-level components are delivered as an updated installer package. These components include Docker, Kubernetes, and the Ops Center.
+* Cluster-level components, which include Docker, Kubernetes, and the Ops Center. These components run on virtual machines or bare-metal servers that act as nodes on the cluster.
++
+Updates are delivered as an updated installer package via the *Runtime Fabric downloads* page in Anypoint Runtime Manager.  The installer package is also referred to as the Runtime Fabric appliance.
 
-* Runtime Fabric components are delivered via Anypoint Runtime Manager in Anypoint Platform. These components include the agent used to connect Runtime Fabric to Anypoint Platform. If an updated application or appliance version is available, a link to the downloads page is displayed in the *Upgrade Available* and *Appliance Version* field, as applicable.
+* Runtime Fabric components, which include the agent used to connect Runtime Fabric to Anypoint Platform.
++
+Updates are delivered via the Runtime Fabric page in Anypoint Runtime Manager. If an updated application or appliance version is available, a link to the downloads page is displayed in the *Upgrade Available* and *Appliance Version* fields, as applicable.
 
-* Node configuration changes are reflected in newer versions of the installer scripts, and are also delivered via the `rtfctl` command-line utility. These changes include adjusting operating system limits and performing jobs to clean up system resources.
+* The `rtfctl` command-line utility, which is used to manage Runtime Fabric and perform the actual upgrades to cluster-level and node configuration components.
++
+Upgrades are delivered as a new binary via the *Runtime Fabric downloads* page in Runtime Manager.
 
-* The `rtfctl` command-line utility upgrades are provided as a new binary that replaces the existing binary. The `rtfctl` utility is used to manage Runtime Fabric locally.
+* Node configuration, which includes scripts for provisioning AWS and Azure infrastructure, preparing each node, bootstrapping a new Runtime Fabric, adjusting operating system limits, and performing jobs to clean up system resources.
++
+Changes are delivered as an updated install script version via the *Runtime Fabric downloads* page in Runtime Manager.
++
+[NOTE]
+If you are upgrading the node configuration with install scripts, the latest version of the cluster-level components (installer package) is automatically downloaded and used.
 
-Upgrades to cluster-level and node configuration components are performed using the `rtfctl` command-line utility.
+It is important to run with the most current versions of the cluster-level (installer package) and `rtfctl` components.
 
-## Downloads page
+== Determine the Component Versions You Are Using
```

Comment:
Like how we're able to combine this due to having support for running a single command to gather the versions of all components.

Consider adjusting the header to make this more apparent -- something like "Determine the versions of each component"

Reply:
Updated - thank you!